collector/README.md

```
@@ -42,27 +42,158 @@ Prepare a clean env with the target framework and nccl lib installed.
+# Collector v2 full run: aggregate base ops plus every model case YAML file.
+# Runs only ops/cases represented in collector v2 YAML.
+python3 collect.py --backend trtllm --model-cases-full
+
+# Raw backend registry run: no collector v2 case plan.
+# Runs every op registered by the backend with each collector's default cases.
+python3 collect.py --backend trtllm
+```
+
+Case files:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)